src/inverseRobot.js

```diff
@@ -25,6 +25,24 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const keys = Object.values(robot);
+  const values = Object.keys(robot);
+
+  for (let i = 0; i < keys.length - 1; i++) {
+    for (let j = i + 1; j < keys.length; j++) {
```

Comment:
nope. too expensive of a calculation. you really need one for loop here and one object